lib/constants/payout-method.js

```diff
@@ -0,0 +1,5 @@
+export const PayoutMethodType = {
+  PAYPAL: 'PAYPAL',
+  // BANK_ACCOUNT: 'BANK_ACCOUNT',
```

Comment:
![](https://media.giphy.com/media/pHb82xtBPfqEg/giphy.gif)

Reply:
:smile: 

Did you merge something similar yet? I can un-comment right?

Reply:
Leave it to me.